apps/src/templates/globalEdition/RegionResetButton/style.scss

```diff
@@ -0,0 +1,5 @@
+#global-edition-region-reset-button-container {
+  & form {
+    all: unset;
+  };
```

Comment:
Don't need the `&` or this last `;` (we probably need some kind of scss linter maybe)